index.yaml

```
@@ -40,6 +40,15 @@ pages:
     file: contributing.md
     description: How to contribute to Ignition.
 releases:
-  - citadel
-  - blueprint
-  - acropolis
+  - name: citadel
+    lts: true
+    eol: false
+    description: Recommended version.
+  - name: blueprint
+    lts: false
+    eol: false
+    description: Works with Ubtuntu 18.04.
```

@nkoenig Louise and I agreed on this description:
```suggestion
    description: Supported May, 2019 to December, 2020
```